eppo_core/src/event_ingestion/event.rs

```diff
@@ -1,11 +1,18 @@
-use crate::timestamp::Timestamp;
 use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+
+use crate::timestamp::Timestamp;
 
+#[serde_as]
```

Comment:
We don't need `serde_as` if we don't use `serde_as` attribute below (we're using `serde(with)`)
```suggestion
```